tools/autograd/derivatives.yaml

```diff
@@ -713,6 +713,19 @@
 - name: special_xlog1py.other_scalar(Tensor self, Scalar other) -> Tensor
   self: grad * log1p(other.toDouble())
 
+- name: special_betainc(Tensor self, Tensor a, Tensor b) -> Tensor
+  self: grad * 1
```

Comment:
no need to do * 1 here, you can just return grad as-is if the function has a Jacobian which is an identity matrix (I don't know the function so maybe that's not true?).

Reply:
Sure. Right now this is a bolierplate. There will be the true grad once implemented.